tests_integration/test_single_transaction_autocommit.py

```diff
@@ -0,0 +1,108 @@
+import uuid
+
+import pytest
+
+
+@pytest.mark.integration
+def test_autocommit_on(runner):
+    result = runner.invoke_with_connection_json(
+        [
+            "sql",
+            "-q",
+            "show parameters like 'autocommit'",
+        ]
+    )
+    assert result.exit_code == 0, result.output
+    assert result.json[0]["key"] == "AUTOCOMMIT"
+    assert result.json[0]["default"] == "true"
+    assert result.json[0]["value"] == "true"
```

Comment:
Those two tests could be merged using `parametrize`, wdyth?

```suggestion
@pytest.mark.integration
@pytest.mark.parametrize("autocommit", [True, False])
def test_autocommit(runner, autocommit):
    cmd = [
        "sql", "-q", "show parameters like 'autocommit'"
    ]
    if autocommit:
        cmd.append("--single-transaction")

    result = runner.invoke_with_connection_json(cmd)
    assert result.exit_code == 0, result.output
    assert result.json[0]["key"] == "AUTOCOMMIT"
    assert result.json[0]["default"] == "true"
    assert result.json[0]["value"] == str(autocommit).lower()
```

Reply:
I prefer to avoid conditional logic in tests.
Also with autocommit disabled we wrap the query in BEGIN/COMMIT so additional logic will be required to handle different statement count in result testing